CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -523,10 +523,11 @@ public function buildQuickForm() {
       $getTaxDetails = FALSE;
       $taxTerm = CRM_Utils_Array::value('tax_term', $invoiceSettings);
```

Comment:
I feel these edits are going to require a lot more testing; multiple line items; online contribution form; w/ sales taxes;
    I would prefer we merge your TaxTerm work in now: either delete this commit - or by replacing it with a less invasive commit -> a TPL edit. All we really want is to change how line.tax_rate is displayed in the LineItem.tpl

`        <td class="right">{$taxTerm} ({$line.tax_rate}%)</td>`